samples/java_layout/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    namespace 'io.customer.android.sample.java_layout'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "io.customer.android.sample.java_layout"
+        minSdk 21
+        targetSdk 33
+        versionCode 1
+        versionName "1.0"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    buildFeatures {
+        viewBinding true
+    }
+}
+
+dependencies {
+    // Updating lifecycle version causes duplicate classes error because a large
+    // number of Lifecycle classes have been converted to Kotlin since 2.6
```

Comment:
we can exclude those? using something like this ?

```
    packagingOptions {
        exclude 'META-INF/lifecycle_runtime.kotlin_module'
        exclude 'META-INF/rxjava.properties'
        exclude 'META-INF/proguard/androidx-annotations.pro'
        exclude 'META-INF/proguard/androidx-core.pro'
    }
```

Reply:
As discussed on slack, [this answer](https://stackoverflow.com/a/75298544) has helped solving this problem. Thanks for sharing 👍🏻